website/docs/getting_started/flashing_firmware/flashing_firmware_cli.md

```diff
@@ -0,0 +1,77 @@
+---
+id: flashing_firmware_cli
+title: Flashing Firmware – CLI
+sidebar_label: Command Line Interface
+slug: /getting_started/flashing_firmware/cli
+---
+
+:::caution
+Be very careful to install the correct load for your board. In particular the popular 'T-BEAM' radio from TTGO is not called 'TTGO-Lora' (that is a different board). So don't install the 'TTGO-Lora' build on a TBEAM, it won't work correctly.
+:::
+
+## Linux
```

Comment:
Is it worth splitting this section up into different pages for each OS? Just thinking this will make it easier for a beginner to work out what they're doing. For example see https://meshtastic.vercel.app/docs/getting_started/firmware/windows

Reply:
Fair point! I actually like that a lot more. I was pretty focused on the methods when I wrote this, but doing an operating system allows for things to be written pretty straight forward.

I'll absolutely make this change.